docs/traffic-policy/macros/index.mdx

```diff
@@ -778,6 +778,32 @@ The following is an example of using `rand.int` with custom values:
 	}}
 />
 
+## Secrets
```

Comment:
@sg-writer Sorry for the churn here. I'm readding this because I realized not documenting it here too actually is kinda weird!